sopel/plugins/rules.py

```diff
@@ -1267,14 +1322,13 @@ def get_rule_regex(self):
         * the rule's name (escaped for regex if needed),
         * all of its aliases (escaped for regex if needed),
 
-        and everything is then given to
-        :func:`sopel.tools.get_action_command_regexp`, which creates a compiled
-        regex to return.
+        to create a compiled regex to return.
         """
         name = [self.escape_name(self._name)]
         aliases = [self.escape_name(alias) for alias in self._aliases]
         pattern = r'|'.join(name + aliases)
-        return tools.get_action_command_regexp(pattern)
+        pattern = self.PATTERN_TEMPLATE.format(command=pattern)
+        return re.compile(pattern, re.IGNORECASE | re.VERBOSE)
```

Comment:
Reusing the name `pattern` like this is a little bit weird, but I won't block on it.

Reply:
I was tempted to rename that as ` base_pattern` or ` command_pattern`, but heh, I'm fine with just `pattern`.